readme.md

```diff
@@ -250,6 +250,39 @@ got.post('google.com', {
 ```
 
 
+## OAuth
+
+You can use the [`oauth-1.0a`](https://www.npmjs.com/package/oauth-1.0a) module to create a signed OAuth request:
+
+```js
+const got = require('got');
+const OAuth = require('oauth-1.0a');
+
+
+const oauth = OAuth({
+  consumer: {
+    public: CONSUMER_KEY,
+    secret: CONSUMER_SECRET
```

Comment:
Unclear where these variables (`CONSUMER_SECRET`) are coming from. Use `process.env.CONSUMER_SECRET` for all of these.